gui/resources.xml

```diff
@@ -597,21 +618,21 @@ Vocabulary
         <object class="GtkAction" id="ResetZoom">
           <property name="icon-name">mypaint-view-100-symbolic</property>
           <property name="label" translatable="yes" context="Menu→View→Reset (labels), Accel Editor (labels)">Reset Zoom</property>
-          <property name="tooltip" translatable="yes" context="Accel Editor (descriptions)">Restore the view’s zoom to its default.</property>
+          <property name="tooltip" translatable="yes" context="Accel Editor (descriptions)">Restore the view’s zoom to 100%.</property>
           <signal name="activate" handler="reset_view_cb"/>
         </object>
       </child>
       <child>
         <object class="GtkAction" id="ResetRotation">
           <property name="label" translatable="yes" context="Menu→View→Reset (labels), Accel Editor (labels)">Reset Rotation</property>
-          <property name="tooltip" translatable="yes" context="Accel Editor (descriptions)">Restore the view’s rotation to 0°</property>
+          <property name="tooltip" translatable="yes" context="Accel Editor (descriptions)">Reset the view’s rotation to 0°</property>
```

Comment:
:+1: I think it was intended as a gloss for then slightly-technical word "Reset", before the accelerator editor came along. Now it's unhelpful wording because it's not clear that it means what the menu calls "Reset".